docs/CI.md

```diff
@@ -0,0 +1,24 @@
+# Containers and Continuous Integration
+
+JSS uses [Travis CI](https://travis-ci.com/about) for PR gating. We use
```

Comment:
How about adding the Travis link to repo instead of about? https://travis-ci.org/dogtagpki/jss